app/templates/main/manageServiceLearningFaculty.html

```diff
@@ -12,26 +17,25 @@
   <div class ="container">
     <a class="skipContent" href="#backButton" tabindex="1" style="top:8%">Click to Skip</a>
     <div class="contributors" align="center">
-      <h2> Manage Service Learning Faculty </h2>
+      <h1 class="text-center mt-5 mb-3">Manage Service Learning Faculty</h1>
   <table class="table table-striped" id="myTable">
-    <thead>
-    <tr class="header">
-    <th scope = "col">Name</th>
-    <th scope = "col">Prev Classes</th>
-    </tr>
-    </thead>
-  <tbody id = "facultyTable">
-  {% for instructor in courseInstructors%}
-  <tr>
-
-    <td> <a href=" /courseManagement" class="link-primary">{{instructor}}</a></td>
-    <td>{{(', '.join(courseInstructors[instructor]))}}</td>
-
-  </tr>
-  {% endfor %}
-  </tbody>
+      <thead>
+        <tr class="header">
+          <th scope = "col">Name</th>
+          <th scope = "col">Prev Classes</th>
+        </tr>
+      </thead>
+    <tbody id = "facultyTable">
+    {% for instructor in courseInstructors%}
+        <tr>
+          <td> <a href=" /courseManagement" class="link-primary">{{instructor}}</a></td>
+          <td>{{(', '.join(courseInstructors[instructor]))}}</td>
+        </tr>

```

Comment:
Do we want usernames or full names here? Right now it will be showing usernames. We probably want full names

Reply:
fixed it